addons/mail/static/src/discuss/call/common/pip_service.js

```diff
@@ -0,0 +1,62 @@
+import { registry } from "@web/core/registry";
+import { Call } from "@mail/discuss/call/common/call";
+import { reactive } from "@odoo/owl";
+
+export const callPipService = {
+    dependencies: ["mail.popout"],
+    /**
+     * @param {import("@web/env").OdooEnv} env
+     * @param {import("services").ServiceFactories} services
+     */
+    start(env, services) {
+        const popoutService = services["mail.popout"];
+        const popout = popoutService.createManager(Symbol("discuss.native.pip"));
+        let pipWindow = null;
+        const state = reactive({
+            active: false,
+        });
+        popout.addHooks(
+            () => {},
+            () => {
+                state.active = false;
+                env.services["discuss.rtc"]?.channel?.openChatWindow();
+            }
+        );
+        function closePip() {
+            state.active = false;
+            pipWindow?.close();
+        }
+        async function openPip() {
+            const rtc = env.services["discuss.rtc"];
+            if (!rtc?.channel) {
+                return;
+            }
+            state.active = true;
+            if (!window.documentPictureInPicture) {
+                return;
+            }
+            pipWindow = await popout.pip(Call, {
+                props: { isPip: true, thread: rtc.channel },
+                options: { height: Math.min(240, window.innerHeight) },
```

Comment:
The inner-code of `pip` can enforce minimal `240`, to not repeat `240` magic value twice.